### PR TITLE
Add STL exporter in binary format to menu bar

### PR DESCRIPTION
**Pull request by hassadee**
This allows users to export STL file in binary format from header menu.

### CODE REVIEW

**Comment by Mugen87** [comment]
For testing: https://raw.githack.com/hassadee/three.js/dev/editor/index.html

**Review by Mugen87 — APPROVED** [review]

**Comment by hassadee** [comment]
> For testing: https://raw.githack.com/hassadee/three.js/dev/editor/index.html

Thanks, and it works! :)

File: editor/js/Menubar.File.js [hunk 1]
```diff
@@ -279,6 +279,20 @@ Menubar.File = function ( editor ) {
...
+	// Export STL (Binary)
+	
+	var option = new UI.Row();
+	option.setClass( 'option' );
+	option.setTextContent( 'Export STL (Binary)' );
+	option.onClick( function () {
+
+		var exporter = new THREE.STLExporter();
+
+		saveString( exporter.parse( editor.scene, { binary: true } ), 'model-binary.stl' );
```
Comment (mrdoob) [inline editor/js/Menubar.File.js]: Shouldn't this use `saveArrayBuffer()` instead?

Comment (hassadee) [reply editor/js/Menubar.File.js]: Oh, yes we should use `saveArrayBuffer()` instead for binary format. I'll change it now. : )

**Review by hassadee — COMMENTED** [review]
Fixed.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**